Modules/_operator.c

```diff
@@ -727,6 +727,20 @@ _operator_is_not_impl(PyObject *module, PyObject *a, PyObject *b)
     return Py_NewRef(result);
 }
 
+/*[clinic input]
+_operator.is_none = _operator.neg
+
+Same as a is None.
+[clinic start generated code]*/
+
+static PyObject *
+_operator_is_none(PyObject *module, PyObject *a)
+/*[clinic end generated code: output=07159cc102261dec input=0448b38af7b8533d]*/
+{
+    PyObject *result = Py_IsNone(a) ? Py_True : Py_False;
+    return Py_NewRef(result);
```

Comment:
No need to newref here. Both `Py_True` and `Py_False` are immortal. So refcounting ops have no effect on them.

Reply:
> No need to newref here. Both `Py_True` and `Py_False` are immortal. So refcounting ops have no effect on them.

That's correct, but if we want to maintain consistency with other functions in this module, then we should keep that.

Reply:
I still agree with @Eclips4. This had been discussed in the previous comments already. Thanks for the heads-up anyway!